src/components/ItemEditorPage/RightPanel/RightPanel.tsx

```diff
@@ -397,7 +428,7 @@ export default class RightPanel extends React.PureComponent<Props, State> {
                           info={t('item_editor.right_panel.replaces_info')}
                           value={replaces}
                           options={this.asCategorySelect(overrideCategories)}
-                          disabled={!canEditItemMetadata}
+                          disabled={!canEditItemMetadata || this.isSkin()}
```

Comment:
Shouldn't the hides selector be disabled as well if it's a skin?

Reply:
No, the creator can choose to hide accessories that are not hidden by default by skins